packages/fxa-admin-panel/package.json

```diff
@@ -54,16 +56,18 @@
     "@types/express": "^4.17.2",
     "@types/helmet": "0.0.45",
     "@types/jest": "^25.1.3",
-    "@types/jsdom": "^16.1.0",
+    "@types/jsdom": "^12.2.4",
```

Comment:
This was apparently a necessary change (for now), as I was not able to `npm run build:server`. [See this thread](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/26338#issuecomment-589523765).